from_cpython/Include/Python.h

```diff
@@ -76,6 +76,7 @@
 #include "descrobject.h"
 #include "warnings.h"
 #include "weakrefobject.h"
+#include "socketmodule.h"
```

Comment:
oh whoops, that was actually already added to the file that needed it (`types.cpp`).  will remove.